scipy/integrate/__init__.py

```diff
@@ -99,6 +107,7 @@
 from ._ivp import (solve_ivp, OdeSolution, DenseOutput,
                    OdeSolver, RK23, RK45, DOP853, Radau, BDF, LSODA)
 from ._quad_vec import quad_vec
+from ._tanhsinh import nsum
```

Comment:
Not sure it's the best place. To consider.

Reply:
At some point it might deserve its own file, but I think it's the best existing file since it's highly dependent on `_tanhsinh` and shares aspects  of the API. Anyway, that's where it is. Maybe move it when it's time to add a `method` argument.